res/values/setup.xml

```diff
@@ -53,6 +53,7 @@
     <bool name="imprint_enabled">false</bool> 
     <bool name="recommend_enabled">true</bool>
     <bool name="feedback_enabled">true</bool>
+    <bool name="logger_enabled">true</bool>
```

Comment:
Thanks, I understand now. I totally oversaw the intention of making the menu available in the official release of the app, not only in debug mode. In this case, having this option here makes a lot of sense.

The point is, I'm not so sure we should have this is the official app. I will follow this up in #1432